db/migrate/20210205150848_update_users.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :kerberos_id, :string, null: false
+    add_column :users, :display_name, :string, null: false
+    add_column :users, :middle_name, :string, null: true
+    add_column :users, :preferred_name, :string, null: true
+    add_column :users, :orcid, :string, null: true
+    
+    add_index :users, [:kerberos_id, :orcid], unique: true
```

Comment:
Your assumption is correct but I'm also not sure what this code is doing so I'll let someone else respond to that part :)

Reply:
My understanding is that it would create two separate indices, which is my intention, but I'm not entirely sure how to confirm that. It's probably not worth the syntactic sugar.

Reply:
Seeing how the schema file changed in response to this last commit makes me think that this updated migration is the one we're hoping for...